molecule/default/verify.yml

```diff
@@ -5,7 +5,13 @@
   any_errors_fatal: true
 
   tasks:
-    - name: "Mock verification task"
-      changed_when: false
+    - name: "Check mountpoint"
       ansible.builtin.command:
-        cmd: "true"
+        cmd: mount -l
```

Comment:
Check out this built-in Ansible test:

https://docs.ansible.com/ansible/latest/collections/ansible/builtin/mount_test.html#ansible-collections-ansible-builtin-mount-test